config/routes.rb

```diff
@@ -7,4 +7,16 @@
     end
   end
   root to: "users#index"
+
+  namespace :api do
+    namespace :v1 do
+      post :auth, to: "authentication#login"
+      resources :users do
+        post :create, to: "users#create"
```

Comment:
- [ ] Your app should have an API endpoint for authentication, both registration, and login, kindly make sure to use the same endpoints as follow so your app is consistent with the requirements:
* Login endpoint - **POST :** `/login`
* Registration endpoint - **POST :** `/register` or `/signup`